build/Dockerfile

```diff
@@ -1,8 +1,9 @@
 # -------- builder stage -------- #
-FROM golang:alpine AS builder
+FROM golang:1.15.5-alpine3.12 AS builder
```

Comment:
Can we keep base image as it is? Updating the golang docker image tag every time can become a maintenance overhead.